plugin/src/templates/edge/utils.ts

```diff
@@ -5,6 +5,19 @@ export interface FetchEventResult {
   waitUntil: Promise<any>
 }
 
+/**
+ * This is how Next handles rewritten URLs.
```

Comment:
[dust] Might be handy to include the same link that was provided in the pull request summary here so it makes it easier to folks to find the relevant code that's being referred to here